db.js

```diff
@@ -570,6 +570,7 @@ exports.init = function (sbot, config) {
 
   function updateIndexes() {
     if (!log.compactionProgress.value.done) return
+    if (abortLogStreamForIndexes) throw Error('updateIndexes() already running')
```

Comment:
I think I would prefer to write this to log and return instead of throwing an error, because nothing is catching those errors so it would be a hard crash.